packages/drafts-realm/opportunity-form.gts

```diff
@@ -0,0 +1,602 @@
+
+    return formattedNumber + '%';
+  }
+
+  <template>
+    <div class='totalCurrencyAmount'>{{this.getFormattedPercentage}}</div>
+
+    <style>
+      .totalCurrencyAmount {
+        color: var(--boxel-dark-teal);
+        font-weight: bold;
+      }
+    </style>
+  </template>
+}
+
+class PercentageField extends FieldDef {
+  @field total = contains(NumberField, {
+    description: `Percentage`,
+  });
+
+  static edit = EditSecForPercentageField;
+  static embedded = EmbdeddedSecForPercentageField;
+}
+
+/* Forecast */
+class EmbeddedSecForForecastField extends Component<typeof ForecastField> {
+  <template>
+    <div class='subject'>{{this.args.model.category}}</div>
+
+    <style>
+      .subject {
+        margin: 0px;
+      }
+    </style>
+  </template>
+}
+
+class EditSecForForecastField extends Component<typeof ForecastField> {
+  @tracked selectedCategory = {
+    name: this.args.model.category || 'None',
+  };
+
+  @tracked categoryPlaceholder = this.args.model.category || 'None';
+
+  @tracked categoryOptions = [
+    { name: 'None' },
+    { name: 'Omitted' },
+    { name: 'Pipeline' },
+    { name: 'Best Case' },
+    { name: 'Commit' },
+    { name: 'Closed' },
+  ] as Array<CategorySignature>;
+
+  @action updateCategory(type: { name: string }) {
+    this.selectedCategory = type;
+    this.args.model.category = type.name;
+  }
+
+  <template>
+    <CardContainer @displayBoundaries={{false}} class='card-container'>
+
+      <BoxelSelect
+        @searchEnabled={{true}}
+        @searchField='name'
+        @placeholder={{this.categoryPlaceholder}}
+        @selected={{this.selectedCategory}}
+        @onChange={{this.updateCategory}}
+        @options={{this.categoryOptions}}
+        class='select'
+        as |item|
+      >
+        <div>{{item.name}}</div>
+      </BoxelSelect>
+
+    </CardContainer>
+
+    <style>
+      .select {
+        padding: var(--boxel-sp-xs);
+        background-color: white;
+      }
+    </style>
+  </template>
+}
+
+class ForecastField extends FieldDef {
+  @field category = contains(StringField, {
+    description: `Selected Category`,
+  });
+
+  static edit = EditSecForForecastField;
+  static embedded = EmbeddedSecForForecastField;
+}
+
+/* Stage */
+class EmbeddedSecForStageField extends Component<typeof StageField> {
+  <template>
+    <div class='subject'>{{this.args.model.category}}</div>
+
+    <style>
+      .subject {
+        margin: 0px;
+      }
+    </style>
+  </template>
+}
+
+class EditSecForStageField extends Component<typeof StageField> {
+  @tracked selectedCategory = {
+    name: this.args.model.category || 'None',
+  };
+
+  @tracked categoryPlaceholder = this.args.model.category || 'None';
+
+  @tracked categoryOptions = [
+    { name: 'None' },
+    { name: 'Qualification' },
+    { name: 'Needs Analysis' },
+    { name: 'Proposal' },
+    { name: 'Negotiation' },
+    { name: 'Closed Won' },
+    { name: 'Closed Lost' },
```

Comment:
May need to model these options data inside card. Lets bring this feedback to Chuan